consensus/src/network.rs

```diff
@@ -132,21 +146,26 @@ pub struct IncomingRandGenRequest {
 
 #[derive(Debug)]
 pub enum IncomingRpcRequest {
-    BlockRetrieval(IncomingBlockRetrievalRequest),
+    /// NOTE: This is being phased out in two releases to accommodate `IncomingBlockRetrievalRequestV2`
+    /// TODO @bchocho @hariria can remove after all nodes upgrade to release with enum BlockRetrievalRequest (not struct)
+    DeprecatedBlockRetrieval(IncomingBlockRetrievalRequest),
```

Comment:
I am wondering if we should append V0 to `IncomingBlockRetrievalRequest` and make it `IncomingBlockRetrievalRequestV0`. That way we can keep the new variant as `BlockRetrieval` instead?

Reply:
Changed to 

```rust
#[derive(Debug)]
pub enum IncomingRpcRequest {
    /// NOTE: This is being phased out in two releases to accommodate `IncomingBlockRetrievalRequestV2`
    /// TODO @bchocho @hariria can remove after all nodes upgrade to release with enum BlockRetrievalRequest (not struct)
    DeprecatedBlockRetrieval(DeprecatedIncomingBlockRetrievalRequest),
    BatchRetrieval(IncomingBatchRetrievalRequest),
    DAGRequest(IncomingDAGRequest),
    CommitRequest(IncomingCommitRequest),
    RandGenRequest(IncomingRandGenRequest),
    BlockRetrieval(IncomingBlockRetrievalRequest),
}
```